docs/src/content/docs/guides/contribute.md

```diff
@@ -10,13 +10,13 @@ sidebar:
 
 You can contribute to this repository in many ways:
 
-🔥 Create a new challenge by following the intructions [here](/guides/create-challenge).
+🔥 Create a new challenge by following the instructions [here](/guides/create-challenge).
 
 🔥 Answer challenges and submit the results. (guide [here](/guides/resolve-challenge)).
 
-🔥 Comment on other's solutions by providing constructive and caring feedback.
+🔥 Comment on others' solutions by providing constructive and caring feedback.
 
-🔥 Correct typos or English mistakes within the documentation.
+🔥 Correct typos within the documentation.
```

Comment:
> Correct typos within the documentation or add new translations.

Maybe add this "add new translations" part?

Reply:
I added a separate line for the translation.  This is a nice addition.

Reply:
Thanks a lot, that's great 👍